lib/sdp/media.ex

```diff
@@ -0,0 +1,162 @@
+defmodule Membrane.Protocol.SDP.Media do
+  @moduledoc """
+  This module represents Media field of SDP.
+
+  For more details please see [RFC4566 Section 5.14](https://tools.ietf.org/html/rfc4566#section-5.14)
+  """
+  use Bunch
+  @enforce_keys [:type, :ports, :protocol, :fmt]
+  defstruct @enforce_keys ++
+              [
```

Comment:
It can be done even better. If you move `connection_data: []` to the last places in list you don't have to specify `title: nil`. For example:

```
[
                :title,
                :encryption,
                bandwidth: [],
                attributes: [],
                connection_data: []
              ]
```

Reply:
It's also OK, but in this case, I think explicit defaults are better if you don't enforce these keys. Anyway, choose what you prefer